sdk/ruby/lib/apicco_sdk.rb

```diff
@@ -0,0 +1,6 @@
+require "active_support/inflector"
```

Comment:
❓ Can we write the SDK without using these dependencies in the SDK since they are quite big?

Reply:
that was my initial attempt but it failed. i will try again.

Reply:
https://github.com/klouvas/apicco/pull/1/commits/c5ba0fa9f525070c82fabbfa953243e0b80730f7